README.md

```diff
@@ -1,27 +1,57 @@
 # PeselGen
+## Overview
+PeselGen is REST webservice for creating random personal data for machine learning with universal random data generator.
```

Comment:
Creating defined data sets, including personal data and medical metrics data for machine learing